### ISSUE TITLE
Issue creating a custom pooling layer

### ISSUE THREAD

**Issue by alexandrudaia** [keras-team/keras#14342]
I am searching a way to build a custom pooling layer.
More exactly I would  whant  instead of getting max or avg  values , I would like to get the sum of probabilities.
This for example  does not help me when I try to add it as lambda layer .
Any suggestions ?

> def trivial_pool(x):
             pool=  K.sum(np.unique(x,return_counts=True)[1], pool_size=(2, 2), strides=(2, 2))
             channel = -1
             return ( pool , axis=channel)

**Comment by sushreebarsa** [comment]
@alexandrudaia Could you please refer to the [link1](https://www.tensorflow.org/tutorials/customization/custom_layers) ,l[ink2](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Lambda) and let us know if it helps ? Thanks !

**Comment by sushreebarsa** [comment]
Closing this issue due to lack of recent activity & assuming this issue as resolved .Please feel free to reopen if you still have concern.Thanks!
